File: MayorMenor/mayormenor.py
num1 = int(input("Ingrese el primer numero: "))
num2 = int(input("Ingrese el segundo numero: "))

if(num1 > num2):
    print("El mayor es: ", num1)
    print("La diferencia entre el numero 1 y numero 2 es: ", num1 - num2)
elif(num1 == num2):
    print("Los numeros son iguales: ")
    print("La diferencia entre el numero 1 y numero 2 es: ", num1 - num2)
else:
    print("El mayor es: ", num2)
    print("La diferencia entre el numero 1 y numero 2 es: ", num2 - num1)

File: MayorMenor/MayorMenor.cs
using System;

namespace MayorMenor
{
    class MayorMenor
    {
        public static void Main(string[] args){
            int num1, num2;
            Console.WriteLine("Ingrese el primer numero: ");
            num1 = int.Parse(Console.ReadLine());
            Console.WriteLine("Ingrese el segundo numero: ");
            num2 = int.Parse(Console.ReadLine());
            if(num1 > num2){
                Console.WriteLine("El numero mayor es: " + num1 + " y la diferencia entre ambos es: "+ (num1-num2));
            }else if(num1 == num2){
                Console.WriteLine("Los numeros son iguales y la diferencia entre ambos es: 0");
            }else{
                Console.WriteLine("El numero mayor es: " + num2 + " y la diferencia entre ambos es: "+ (num2-num1));
            }

        }
    }
}